### PR TITLE
oml.0.0.1 - via opam-publish

### PR DESCRIPTION
**Pull request by rleonid**
OCaml Math (and Statiscs) Library

Common algorithms and procedures to perform exploratory data analysis,
machine learning and mathematical simulations.

---
* Homepage: https://github.com/hammerlab/oml/
* Source repo: https://github.com/hammerlab/oml/
* Bug tracker: https://github.com/hammerlab/oml/issues

---

Pull-request generated by opam-publish v0.3.0

### CODE REVIEW

File: packages/oml/oml.0.0.1/descr [hunk 1]
```diff
@@ -0,0 +1,4 @@
+OCaml Math (and Statiscs) Library
```
Comment (dsheets) [inline packages/oml/oml.0.0.1/descr]: I think you accidentally a whole syllable.

File: packages/oml/oml.0.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/hammerlab/oml/"
+dev-repo: "https://github.com/hammerlab/oml/"
```
Comment (dsheets) [inline packages/oml/oml.0.0.1/opam]: This needs an `oml.git` where there is `oml/`.

File: packages/oml/oml.0.0.1/descr (hunk 1 above)
Comment (rleonid) [inline packages/oml/oml.0.0.1/descr]: Noted :)

File: packages/oml/oml.0.0.1/opam (hunk 2 above)
Comment (rleonid) [inline packages/oml/oml.0.0.1/opam]: Thank you. I wish `opam-publish` did this for me as I always forget.

File: packages/oml/oml.0.0.1/descr [hunk 3]
```diff
@@ -0,0 +1,4 @@
+beginnings of an OCaml Math (and statiscs) Library
```
Comment (dsheets) [inline packages/oml/oml.0.0.1/descr]: I assume `statiscs` is supposed to be `statistics` so it is still missing a syllable.

Comment (rleonid) [inline packages/oml/oml.0.0.1/descr]: Thank you

File: packages/oml/oml.0.0.1/opam (hunk 2 above)
Comment (dsheets) [inline packages/oml/oml.0.0.1/opam]: ocaml/opam#2240

**Comment by rleonid** [comment]
Wow, what is going on in that failed build? `|>` isn't available?

**Comment by rleonid** [comment]
Updated constraint to >= 4.01. I'll change Oml, in the future to be backwards compatible.

**Pull request merged**